CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# Contributing
+
+Thank you for wanting to help us!
+Read this document for information on how to get started.
+
+## General overview of project
+
+We use [Next.js](https://nextjs.org/) to generate the static website.
+All our website styling is done with the [Chakra UI](https://chakra-ui.com/) framework.
+
+## What you'll need to know to be able to help
+
+You'll need a basic understanding of HTML/CSS to be able to help with the code.
+It's even better if you know a bit of React and the basics of TypeScript.
+
+## Software requirements
+
+- Git
+- Latest version of Node.js 14
+- Yarn v1 package manager
+
+This guide assumes you'll run the code on your workstation directly.
+
+### Where to get Git
+
+The [git-scm.com](https://git-scm.com/) site has links to download Git for your operating system.
+
+### Where to get Node.js
+
+If you use Linux, use your operating system's package manager (`dnf`, `apt` or similar) to install the latest version of Node 14.
+If you use Ubuntu, you can also get a snap of the Node 14 program.
+
+If you use Windows or macOS, you can grab binaries from the [Node.js offical website](https://nodejs.org/).
+
+### Where to get Yarn
+
+Once you have Node 14 installed, you can install Yarn v1 with node's `npm` package manager:
+
+```bash
+$ npm install -g yarn
+```
+
+## Make a fork
+
+Use the GitHub web interface to create your own fork of the Octoclairvoyant repository.
+
+Do not put your own work on your forks `main` branch.
+Create a new feature branch for each bit of work!
+
+## Installing development dependencies
+
+We use Yarn v1 to manage our dependencies.
+Run the `yarn` command in your own Octoclairvoyant directory to install all dependencies:
+
+```bash
+$ yarn
+```
+
+## Development process
+
+You're ready to start work now.
+We recommend you follow this process:
+
+1. Create a feature branch
+1. Start the development server with `yarn start`
+1. Make improvements
+1. Put chunks of work in a commit (the Husky program will run some checks)
+1. Run `yarn smoketest` to confirm you're not breaking anything critical
+1. Create pull request
+
+## Query string to check comparator output
+
+Use the following query string to check the comparator output: INSERT PROPER LINK HERE
```

Comment:
@Belco90 Can you suggest a realistic query string we should use here?

Reply:
I tend to use this comparison for checking everything is working as expected: `?repo=testing-library%2Fdom-testing-library&from=v6.16.0&to=v8.1.0`

It includes major, minor, and patch semver, code of blocks, diffs, commits and PRs references, different text emphasis... I think this is a good candidate for what you are looking for.